kube/src/api/resource.rs

```diff
@@ -101,6 +101,12 @@ impl Resource {
         if let Some(labels) = &lp.label_selector {
             qp.append_pair("labelSelector", &labels);
         }
+        if let Some(limit) = &lp.limit {
+            qp.append_pair("limit", &limit.to_string());
```

Comment:
i don't think you need to turn this into a string again here ^. you already have an `Option<String>`

Reply:
nevermind, you do, it's a uint :D